src/components/navigation/PewPewNavBar.vue

```diff
@@ -23,13 +23,13 @@
         :href="r.route"
         target="_blank"
       >
-        <div class="pewpew__navlink">
+        <div class="pewpew__navlink text-white">
           <ccu-icon :linked="true" v-bind="r.iconProps" />
           {{ r.title }}
         </div>
       </a>
       <router-link v-else :key="r.title" :to="r.routeProps">
-        <div class="pewpew__navlink">
+        <div class="pewpew__navlink text-white">
```

Comment:
Check my comments below. This class should be added to `pewpew__navlink` class

Reply:
done